src/autocapture-utils.ts

```diff
@@ -338,7 +338,7 @@ export function getDirectAndNestedSpanText(target: Element): string {
  */
 export function getNestedSpanText(target: Element): string {
     let text = ''
-    if (target.children?.length > 0) {
+    if (target && target.children?.length > 0) {
```

Comment:
I think it's a nice balance between the legible version and the paranoid version.

It's a really tricky balance. If we never trust TS then what's the point of using it, but it's JS that runs in the browser and we know that'll do the stupidest thing at least 50% of the time 🤣

Reply:
💯